lib/diagnostic.rb

```diff
@@ -1,44 +1,53 @@
 # frozen_string_literal: true
-
+list = 1..10
 # Ruby enumerable diagnostic
 class Diagnostic
   # Question 1
   # Write ruby code to return the longest entry in the list supplied.
   # Assume each entry in the list implements a length method that returns
   # a Fixnum.
   def longest(list)
-    # Your code goes here
+    list.max
```

Comment:
Using `.max` we would usually use a block so it knows how to compare each element:

`list.max { |a, b}| a.length <=> b.length }`